Description: Add openstack_auth in INSTALLED_APPS
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-01-11

--- horizon-9.0.0~b2+2016.02.18.git.cda9604792.orig/horizon/test/settings.py
+++ horizon-9.0.0~b2+2016.02.18.git.cda9604792/horizon/test/settings.py
@@ -59,7 +59,8 @@ INSTALLED_APPS = (
     'horizon',
     'horizon.test',
     'horizon.test.test_dashboards.cats',
-    'horizon.test.test_dashboards.dogs'
+    'horizon.test.test_dashboards.dogs',
+    'openstack_auth'
 )
 
 MIDDLEWARE_CLASSES = (
