Description: [Django 1.10] openstack_dashboard.test.settings
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-21

--- horizon-10.0.0~b2.orig/openstack_dashboard/test/settings.py
+++ horizon-10.0.0~b2/openstack_dashboard/test/settings.py
@@ -242,3 +242,23 @@ REST_API_SETTING_2 = 'bar'
 REST_API_SECURITY = 'SECURITY'
 REST_API_REQUIRED_SETTINGS = ['REST_API_SETTING_1']
 REST_API_ADDITIONAL_SETTINGS = ['REST_API_SETTING_2']
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'APP_DIRS': True,
+        'DIRS': [os.path.join(TEST_DIR, 'templates'), ],
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.request',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.contrib.messages.context_processors.messages',
+                'horizon.context_processors.horizon',
+                'openstack_dashboard.context_processors.openstack',
+            ],
+        },
+    },
+]
