Description: [Django 1.10] openstack_dashboard.test.settings
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-08-04

Index: horizon/openstack_dashboard/test/settings.py
===================================================================
--- horizon.orig/openstack_dashboard/test/settings.py
+++ horizon/openstack_dashboard/test/settings.py
@@ -258,3 +258,30 @@ REST_API_SETTING_2 = 'bar'
 REST_API_SECURITY = 'SECURITY'
 REST_API_REQUIRED_SETTINGS = ['REST_API_SETTING_1']
 REST_API_ADDITIONAL_SETTINGS = ['REST_API_SETTING_2']
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [os.path.join(TEST_DIR, 'templates'), ],
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.request',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.contrib.messages.context_processors.messages',
+                'horizon.context_processors.horizon',
+                'openstack_dashboard.context_processors.openstack',
+            ],
+            'loaders': [
+                ('django.template.loaders.cached.Loader', [
+                    'django.template.loaders.filesystem.Loader',
+                    'django.template.loaders.app_directories.Loader',
+                    'horizon.themes.ThemeTemplateLoader',
+                    'horizon.loaders.TemplateLoader',
+                ]),
+            ],
+        },
+    },
+]
