Description: [Django 1.10] Defines TEMPLATES
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-21

Index: deb-horizon/horizon/test/settings.py
===================================================================
--- deb-horizon.orig/horizon/test/settings.py
+++ deb-horizon/horizon/test/settings.py
@@ -74,11 +74,11 @@ MIDDLEWARE_CLASSES = (
 )
 
 TEMPLATE_CONTEXT_PROCESSORS = (
-    'django.core.context_processors.debug',
-    'django.core.context_processors.i18n',
-    'django.core.context_processors.request',
-    'django.core.context_processors.media',
-    'django.core.context_processors.static',
+    'django.template.context_processors.debug',
+    'django.template.context_processors.i18n',
+    'django.template.context_processors.request',
+    'django.template.context_processors.media',
+    'django.template.context_processors.static',
     'django.contrib.messages.context_processors.messages',
     'horizon.context_processors.horizon')
 
@@ -172,3 +172,26 @@ LOGGING = {
         }
     }
 }
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [ os.path.join( os.path.dirname(os.path.abspath(__file__)), 'tests', 'templates'), ],
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.request',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.contrib.messages.context_processors.messages',
+                'horizon.context_processors.horizon'
+            ],
+            'loaders': [
+                'django.template.loaders.filesystem.Loader',
+                'django.template.loaders.app_directories.Loader',
+                'horizon.loaders.TemplateLoader'
+            ],
+        },
+    },
+]
