Description: Fixes version detection when building docs
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-05-17

--- horizon-2013.1.1.orig/openstack_dashboard/openstack/common/version.py
+++ horizon-2013.1.1/openstack_dashboard/openstack/common/version.py
@@ -19,6 +19,7 @@ Utilities for consuming the version from
 """
 
 import pkg_resources
+import os
 
 
 class VersionInfo(object):
@@ -54,6 +55,9 @@ class VersionInfo(object):
         if self.release is None:
             self.release = self._get_version_from_pkg_resources()
 
+        if self.release is None:
+            self.release = os.environ.get("OSLO_PACKAGE_VERSION", None)
+
         return self.release
 
     def version_string(self):
