Index: horizon-2012.2/run_tests.sh
===================================================================
--- horizon-2012.2.orig/run_tests.sh	2012-08-23 18:52:34.891973319 -0700
+++ horizon-2012.2/run_tests.sh	2012-08-23 18:53:15.879971813 -0700
@@ -274,25 +274,25 @@
   if [ "$NOSE_WITH_HTML_OUTPUT" = '1' ]; then
     export NOSE_HTML_OUT_FILE='horizon_nose_results.html'
   fi
-  ${command_wrapper} coverage erase
-  ${command_wrapper} coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
+  ${command_wrapper} python-coverage erase
+  ${command_wrapper} python-coverage run -p $root/manage.py test horizon --settings=horizon.tests.testsettings $testargs
   # get results of the Horizon tests
   HORIZON_RESULT=$?
 
-  echo "Running openstack_dashboard tests"
-  export NOSE_XUNIT_FILE=openstack_dashboard/nosetests.xml
-  if [ "$NOSE_WITH_HTML_OUTPUT" = '1' ]; then
-    export NOSE_HTML_OUT_FILE='dashboard_nose_results.html'
-  fi
-  ${command_wrapper} coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
-  # get results of the openstack_dashboard tests
-  DASHBOARD_RESULT=$?
+#  echo "Running openstack_dashboard tests"
+#  export NOSE_XUNIT_FILE=openstack_dashboard/nosetests.xml
+#  if [ "$NOSE_WITH_HTML_OUTPUT" = '1' ]; then
+#    export NOSE_HTML_OUT_FILE='dashboard_nose_results.html'
+#  fi
+#  ${command_wrapper} python-coverage run -p $root/manage.py test openstack_dashboard --settings=openstack_dashboard.test.settings $testargs
+#  # get results of the openstack_dashboard tests
+#  DASHBOARD_RESULT=$?
 
   if [ $with_coverage -eq 1 ]; then
     echo "Generating coverage reports"
-    ${command_wrapper} coverage combine
-    ${command_wrapper} coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
-    ${command_wrapper} coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
+    ${command_wrapper} python-coverage combine
+    ${command_wrapper} python-coverage xml -i --omit='/usr*,setup.py,*egg*,.venv/*'
+    ${command_wrapper} python-coverage html -i --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
   fi
   # Remove the leftover coverage files from the -p flag earlier.
   rm -f .coverage.*
Index: horizon-2012.2/horizon/tests/testsettings.py
===================================================================
--- horizon-2012.2.orig/horizon/tests/testsettings.py	2012-08-23 18:52:34.647973328 -0700
+++ horizon-2012.2/horizon/tests/testsettings.py	2012-08-23 18:54:36.615968847 -0700
@@ -98,7 +98,6 @@
 TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
 NOSE_ARGS = ['--nocapture',
              '--nologcapture',
-             '--exclude-dir=horizon/conf/',
              '--cover-package=horizon',
              '--cover-inclusive']
 
