Description: Fixed horizon MANIFEST.in
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-26

--- horizon-8.0.0~rc1.orig/MANIFEST.in
+++ horizon-8.0.0~rc1/MANIFEST.in
@@ -1,6 +1,6 @@
-recursive-include doc *.py *.rst *.scss *.js *.html *.conf *.jpg *.gif *.png
-recursive-include horizon *.html *.scss *.js *.csv *.template *.tmpl *.mo *.po
-recursive-include openstack_dashboard *.html *.js *.scss *.mo *.po *.example *.eot *.svg *.ttf *.woff *.png *.ico *.wsgi *.gif *.csv *.template
+recursive-include doc *
+recursive-include horizon *
+recursive-include openstack_dashboard *
 recursive-include tools *.py *.sh
 
 include AUTHORS
