--- a/setup.py
+++ b/setup.py
@@ -36,6 +36,7 @@
       author='Devin Carlen',
       author_email='devin.carlen@gmail.com',
       packages=find_packages(),
+      include_package_data=True,
       package_data={'horizon': [s[len('horizon/'):] for s in
                                 findall('horizon/templates') \
                                 + findall('horizon/dashboards/nova/templates') \
--- /dev/null
+++ b/MANIFEST.in
@@ -0,0 +1 @@
+recursive-include horizon *.html *.js *.csv *.template *.mo
